tests/cython/bindlink/test_functions.py

```diff
@@ -1,18 +1,20 @@
-# 
+#
 # Test Functions
 
 from opencog.atomspace import types, Atom, TruthValue
 from opencog.type_constructors import *
```

Comment:
This import also seems not to be needed.  Also, if I understand properly, I guess that `opencog.type_constructors` exports both `ATOMSPACE` and also `atomspace` both of which are set to the same value!???

Reply:
It is needed to import type constructors